src/logger.teams.ts

```diff
@@ -30,3 +30,45 @@ export const teams = {
 		font: '#FFFFFF',
 	},
 };
+
+export const generateSvg = (): string => {
```

Comment:
it might be clearer to move this to a separate file. that way we keep the implementation and documentation processes distinct

Reply:
actually why not put this in `generate-svg.sh`?

Reply:
I wanted to have a way to test this, actually. Maybe I can make that a JS file and node exclusive. It's probably a 0-10ms job anyways.